_nodes/0001.md

```diff
@@ -5,7 +5,7 @@ macrolib: topos
 
 Let $E$ be displayed over $B$, and let $f:x\to y \in B$; a morphism
 $\bar{f}:\bar{x}\to\Sub{f} \bar{y}$ in $E$ is called *cartesian* over $f$ when for
-any $m:u\to x$ and $\bar{h}:\bar{u}\to\Sub{u;f} \bar{y}$ there exists a unique
+any $m:u\to x$ and $\bar{h}:\bar{u}\to\Sub{m;f} \bar{y}$ there exists a unique
```

Comment:
somehow this typo has been here all along :laughing:

Reply:
🤦🏼